packages/js/package.json

```diff
@@ -42,14 +42,8 @@
     "@babel/core": "^7.24.0",
```

Comment:
Can you please recheck what packages we need; I think we don't need the babel packages. I think we only need `vite`, `terser` & `vite-plugn-dts`